README.md

```diff
@@ -151,8 +166,13 @@ There are five services defined in the docker compose
 - The file `collector-config.yaml` that has all the configurations regarding receivers, processors, and exporters is mounted as a volume inside of the container.
 
 ### jaeger service
-- The servcice used `jaegertracing/all-in-one:1.38.1` image which is a all in one image which is suitable for local development.
+- The service used `jaegertracing/all-in-one:1.38.1` image which is a all in one image which is suitable for local development.
 - Exposes the UI for visualization in port 16686.
+
+### prometheus service
+- The service use `prom/prometheus:v2.43.0` image. 
```

Comment:
```suggestion
- The service uses `prom/prometheus:v2.43.0` image. 
```

In general though, do we really need to hardcode versions of docker images (or anything really) in the docs? I mean this is stuff that dependabot will update automatically, but surely we can't keep updating the documentation whenever one version changes

Reply:
Like above I tried to be consistent